fhir_zulip_nlp/fhir_zulip_nlp.py

```diff
@@ -27,7 +27,7 @@
     'zuliprc_path': os.path.join(PROJECT_DIR, 'zuliprc'),  # rc = "runtime config"
     'outpath': os.path.join(PROJECT_DIR, 'zulip_report.csv')
 }
-
+CLIENT = zulip.Client(config_file=CONFIG['zuliprc_path'])
 
 # Functions
 '''
```

Comment:
@rohaher I was gonna leave this but I got tired of scrolling around so much:

One thing I would recommend for large code snippets like this that you might want to go and look at: (a) you could look at a previous commit, or (b) easier I find, you can copy paste the snippet or an old version of your script into a python file and then put it into a folder that is `.gitignore`d. Some folders in the current `.gitignore` that you could use for that are: `_archive`, `dev/`, `env/`, `venv/`, `ignored/` and a few more.
```py
'''
def data(key):
    client = zulip.Client(config_file=CONFIG['zuliprc_path'])
    vals = {}
    mess = get_messages_and_topics(key)

    vals['name'] = key
    vals['number of occurances']=len(mess)


    newest = {
        "anchor": "newest",
        "num_before": 1,
        "num_after": 0,
        "narrow": [
            {"operator": "stream", "operand": "terminology"},
            {"operator": "search", "operand": key},
            ],
        }
    result_new = client.get_messages(newest)
    datenew = result_new['messages'][0]['timestamp']
    datenew1 = datetime.utcfromtimestamp(datenew).strftime('%Y-%m-%d %H:%M:%S')
    vals['newest time']=datenew1

    oldest = {
        "anchor": "oldest",
        "num_before": 0,
        "num_after": 1,
        "narrow": [
            {"operator": "stream", "operand": "terminology"},
            {"operator": "search", "operand": key},
            ],
        }
    result_old = client.get_messages(oldest)
    dateold = result_old['messages'][0]['timestamp']
    dateold1 = datetime.utcfromtimestamp(dateold).strftime('%Y-%m-%d %H:%M:%S')
    vals['oldest time']=dateold1

    return vals
'''
```